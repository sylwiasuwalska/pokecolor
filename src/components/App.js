import React from "react";
import "bootstrap/dist/css/bootstrap.min.css";
import Store from "./Store";
import Header from "./Header";
import PokeContainer from "./PokeContainer";

import "../styles/App.css";
import ButtonContainer from "./ButtonContainer";

function App() {
  return (
    <Store>
      <div className="App">
        <Header />
        <ButtonContainer />
        <PokeContainer />
      </div>
    </Store>
  );
}

export default App;
