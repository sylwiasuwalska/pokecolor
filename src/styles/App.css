.App {
  text-align: center;
}

.container {
  max-width: 90%;
}