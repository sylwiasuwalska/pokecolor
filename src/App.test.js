import React from "react";
import { render, cleanup } from "@testing-library/react";
import "@testing-library/jest-dom";
import App from "./components/App";

afterEach(cleanup);

test("should take a snapshot of App component", () => {
  const { asFragment } = render(<App />);

  expect(asFragment(<App />)).toMatchSnapshot();
});
