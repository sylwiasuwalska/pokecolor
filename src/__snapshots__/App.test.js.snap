// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot of App component 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <h1>
      PoKePedia
    </h1>
    <div
      class="buttonContainer"
    >
      <h4>
        Choose color of Pokemons you want to see:
      </h4>
      <div>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(4, 0, 0); border-color: #040000; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          black 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(27, 52, 97); border-color: #1b3461; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          blue 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(97, 38, 85); border-color: #612655; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          purple 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(195, 23, 54); border-color: #c31736; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          red 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(101, 67, 33); border-color: #654321; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          brown 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(53, 82, 60); border-color: #35523c; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          green 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(98, 100, 102); border-color: #626466; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          grey 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(228, 205, 5); border-color: #e4cd05; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          yellow 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(231, 179, 167); border-color: #e7b3a7; border-radius: 0; color: rgb(255, 255, 255);"
          type="button"
        >
          pink 
        </button>
        <button
          class="btn btn-primary btn-lg"
          style="background-color: rgb(250, 244, 238); border-color: #faf4ee; border-radius: 0; color: rgb(98, 100, 102);"
          type="button"
        >
          white 
        </button>
      </div>
    </div>
    <div
      class="dataPreparing"
    >
      <img
        alt="ball"
        height="100px"
        src="ring.svg"
        width="100px"
      />
      <p>
        Preparing your data. Please wait.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
